services/update-bps/websiteTexts.js

```diff
@@ -0,0 +1,335 @@
+    }
+  },
+  {
+    bp: "eostribeprod",
+    general_info: {
+      websiteText:
+        "EOS Tribe, LLC is a Blockchain company that designs and develops technological solutions for the latest Blockchain smart platforms to solve challenges for real World businesses and serve as catalyst for mass adoption of Blockchain technology Worldwide."
+    }
+  },
+  {
+    bp: "eosyskoreabp",
+    general_info: {
+      websiteText: ""
+    }
+  },
+  {
+    bp: "eosgenblockp",
+    general_info: {
+      websiteText:
+        "A mining as a service company who built the first mining farm in Eastern Europe. Since our founding, we have grown tremendously and a lot has happened, but one thing remains constant: We are all strong believers in the future of digital currencies and we love being part of this growing community."
+    }
+  },
+  {
+    bp: "eosiomeetone",
+    general_info: {
+      websiteText:
+        "MEET.ONE Chain is designed to facilitate the liquidity of everyone’s values，lower the threshold and cost of access to EOSIO layer. Meanwhile, MEET.ONE Chain is committed to make a joint effort with Block Producers and developers, aiming to promote the development of trustworthy Internet and utilize blockchain technology to empower Internet."
+    }
+  },
+  {
+    bp: "eosrapidprod",
+    general_info: {
+      websiteText:
+        "EOS Rapid is a next generation block producer candidate looking to provide both transparency and performance to the EOS community."
+    }
+  },
+  {
+    bp: "eoslaomaocom",
+    general_info: {
+      websiteText:
+        "EOSLaoMao is an EOS Block Producer dedicated to maintaining a fair and stable BP node. We aim to maximize the potential of EOS with the dynamic community around the world."
+    }
+  },
+  {
+    bp: "eosliquideos",
+    general_info: {
+      websiteText:
+        "LiquidEOS is built by passionate blockchain entrepreneurs who believe that the EOS blockchain is pushing the envelope of innovation and will usher in an era of mass user adoption of decentralized applications. We are an R&D center, an EOS Block Producer, & a think tank that’s dedicated to enhancing the EOS network through new ideas, tools, & content. We develop critical EOS infrastructure, build scalable decentralized technologies & engage the global EOS ecosystem through a range of community building initiatives."
+    }
+  }
+];
+
+const massive = require("massive");
+const dbConfig = require("./dbConfig");
+
+const updateGeneralInfo = bpInfo => {
+  console.info("processing:\n", JSON.stringify({ bpInfo }, null, 2));
+  massive(dbConfig).then(db => {
+    db.producers.save(bpInfo, function(err, res) {
+      if (err) {
+        console.log(err);
+        return;
+      }
+      console.log("\nPOSTGRES UPDATED!!\n");
+    });
+  });
+};
+
+const updateBps = bpInfoObj => {
+  if (Array.isArray(bpInfoObj)) {
+    for (var i = 0; i < bpInfoObj.length; i++) {
+      updateGeneralInfo(bpInfoObj[i]);
```

Comment:
@tetogomez  I have a problem here because this for loop opens too many conections to the database. any more suggestions